ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -491,20 +498,23 @@ private class JdbcLedgerDao(
       transaction: CommittedTransaction,
       divulged: Iterable[DivulgedContract],
   )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
-    dbDispatcher
-      .executeSql(metrics.daml.index.db.storeTransactionDbMetrics) { implicit conn =>
-        validate(ledgerEffectiveTime, transaction, divulged) match {
-          case None =>
-            preparedInsert.writeState(metrics)
-            preparedInsert.writeEvents(metrics)
-            insertCompletions(submitterInfo, transactionId, recordTime, offsetStep)
-          case Some(error) =>
-            submitterInfo.foreach(handleError(offsetStep.offset, _, recordTime, error))
-        }
+    withEnrichedLoggingContext(Logging.transactionId(transactionId)) { implicit loggingContext =>
```

Comment:
This message looks unnecessarily verbose in the logfile and every field is prefixed with `update`:
```
updateCommandId=PRPruneQueryEmptyRangeOk-alpha-39f3ccbfdb89-command-220, updateLedgerTime=2021-02-03T15:58:31.515Z, updateOffset=0000000000000a190000000000000000, updateSubmitter=[PRPruneQueryEmptyRangeOk-alpha-39f3ccbfdb89-party-0], transactionId=0A2463633665333436652D373965652D333034322D613862642D653164633433626661636463, updateTransactionId=0A2463633665333436652D373965652D333034322D613862642D653164633433626661636463})
```
Would it be possible to dial it down a bit?

Reply:
I would recommend keeping it as is. Consider that you only see this when using a console logger, in production you should use structured logging and this would not be duplicated in the message.

Reply:
but why do we have the prefix, i.e. updateCommandId vs commandId everywhere else, and why do we have transactionId and updateTransactionId both with the same content?

Reply:
Had a look at the code, the transaction identifier is injected in the context upstream, as soon as it becomes available, using `loggingContextFor` in the trait `com.daml.platform.indexer.ExecuteUpdate`. I believe there's a number of keys that is being inserted twice.

Reply:
Same comment as [above](https://github.com/digital-asset/daml/pull/8676#discussion_r571638038).